gatsby-node.js

```diff
@@ -3,11 +3,14 @@
 const path = require('path');
 const createSlug = require('./util-node/createSlug');
 
+const BLOG_POST_FILENAME_REGEX = /([0-9]+)-([0-9]+)-([0-9]+)-(.+)\.md$/;
```

Comment:
I don't think this will be needed if we move the date from the file name to the front matter.